index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Samuel Felice | Full Stack Developer</title>
+</head>
+<body>
+    <header>
+        <span class="logo">My Logo</span> 
+        <div class="nav-icon">
+          <div class="icon"></div>
+          <div class="icon"></div>          
+          <div class="icon"></div>
+    </div>
+    </header>
+
+    <main>
+        <h1 class="primary-heading">I'm Samuel Glad to see you!</h1>
+
+        <p class="introductory-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <span class="lets-connect">LET’S CONNECT</span>

```

Comment:
- [ ] [REQUIRED] The subtitle should use a header tag `hx` (examples: `<h1>`,`<h2>`,`<h3>`, etc...). please, replace the `<span>` with a heading tag ♻️ .